src/domain/registration/form/RegistrationForm.tsx

```diff
@@ -114,8 +113,9 @@ const RegistrationForm: FunctionComponent<Props> = ({
                 name="guardian.email"
                 label={t('registration.form.guardian.email.input.label')}
                 onChange={handleChange}
-                value={values.guardian.email}
+                value={values.guardian.email || tunnistamoProfile.email}
```

Comment:
Fill user data in here, no need extra type

Reply:
but how are we able to know what the user provided if they want to replace the info from tunnistamo?

Reply:
Hm, I guess we can use tunnistamoProfile.email here since it is not editable

Reply:
and lessee